### PR TITLE
Lego 3477 modal focus trappen oppdateres ikke nar man endrer innholdet live

### PR DESCRIPTION
**Pull request by TrygveUrdahl**
# PR Checklist
https://elvia.atlassian.net/wiki/spaces/TEAMATOM/pages/10427498683/Review+prosess

- [ ] Bumpet package?
- [ ] Updated changelog?
- [ ] Correct date in changelog?

## Describe PR briefly:
https://elvia.atlassian.net/browse/LEGO-3477

### CODE REVIEW

File: packages/components/components/elvis-toolbox/src/hooks/useFocusTrap.ts [hunk 1]
```diff
@@ -65,15 +62,21 @@ export const useFocusTrap = (): {
         lastItem.addEventListener('keydown', handleLastItemTab);
       }
     });
+
+    mo = new MutationObserver(() => {
+      releaseFocusTrap();
+      trapFocus(focusTrapContainer);
+    });
```
Comment (TomSchrier) [inline packages/components/components/elvis-toolbox/src/hooks/useFocusTrap.ts:69]: This assumes that all the content is new. If we had a list of buttons and later added a new button, would we have to TAB through the whole list again? 🤔

Comment (TrygveUrdahl) [reply packages/components/components/elvis-toolbox/src/hooks/useFocusTrap.ts:69]: Hmm thats true, this will reset the focus 🤔 That's not ideal

Comment (TrygveUrdahl) [reply packages/components/components/elvis-toolbox/src/hooks/useFocusTrap.ts]: I've now addressed this issue

**Review by TomSchrier — APPROVED** [review]
Nice! LGTM 🍮

**Pull request merged**